setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+setup(name='Protego',
+      version='0.1dev',
```

Comment:
I think you can use `0.1` or even `1.0` for the first version, to keep the version number numeric.